scripts/migrate-files-to-s3.py

```diff
@@ -0,0 +1,92 @@
+# pylint: disable=invalid-name
+import io
+from server.models import *  # pylint: disable=wildcard-import
+from server.database import db_session
+from server.util.file import store_file, timestamp_filename
+
+
+def migrate_file(file, path):
+    if not file.storage_path:
+        storage_path = store_file(io.BytesIO(file.contents.encode("utf-8")), path)
+        file.storage_path = storage_path
+        return storage_path
+    else:
+        print("Already migrated")
```

Comment:
suggestion: what if this pulled the file from S3 and compared content? that way the script could be used to "check" that everything worked well.

Reply:
sure! will add